src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -447,7 +447,7 @@ The following example will split up the default group Sentry would create (repre
 function makeRequest(path, options) {
     return fetch(path, options).catch(function(err) {
         Sentry.withScope(function(scope) {
-          scope.setFingerprint(['{{default}}', path]);
+          scope.setFingerprint(['{{{{default}}}}', path]);
```

Comment:
```suggestion
          scope.setFingerprint(['{{ "{{default"}}}}', path]);
```

Reply:
You suggestion looks like this for me:

<img width="767" alt="Screen Shot 2020-05-28 at 8 50 18 PM" src="https://user-images.githubusercontent.com/139499/83208883-de60c580-a124-11ea-9296-f556315affed.png">